active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <TextBlock>Hello</TextBlock>
+</Window>
+```
+
+with corresponding code-behind:
+
+```CS
+public partial class MainWindow : Window
+{
+  public MainWindow()
+  {
+	InitializeComponent();
+  }
+}
+```
+
+and then you can create/activate that custom window from code:
+
+```CS
+MainWindow window = new MainWindow();
+window.Activate();
+
+```
+
+> Spec note  
+No <Window.Content> tag is required because the Window.Content property is updating in this spec to become the
+[ContentPropertyAttribute](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Markup.ContentPropertyAttribute).
+
+In a UWP app any UI thread already has a Window on it, which you can retrieve using the static 
+[Window.Current](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Window.Current) 
+property. You can create additional windows by creating additional 
+[CoreApplicationViews](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationView), 
+which are always created on a new thread, and which automatically create the following for the new thread: 
+[ApplicationView](https://docs.microsoft.com/uwp/api/Windows.UI.ViewManagement.ApplicationView), 
+[CoreWindow](https://docs.microsoft.com/uwp/api/Windows.UI.Core.CoreWindow), 
+and the Xaml [Window](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Window).
+
+# Create a new Window on a new thread
+
+This creates a new Xaml Window on a new thread in a UWP app:
+
+```CS
+_ = CoreApplication.CreateNewView().DispatcherQueue.TryEnqueue(() =>
+{
+    // This code runs on the new thread
+    Window.Current.Content = new TextBlock() { Text = "Hello" };
+    Window.Current.Activate();
+});
+```
+
+To create a new Window on a new thread in a Desktop app, create the thread first:
+
+```CS
+var thread = new Thread(() =>
+{
+    Window window = new Window();
+    window.Content = new TextBlock() { Text = "Hello" };
+    window.Activate();
+    window.Closed += (_, __) => w.DispatcherQueue.InvokeShutdown();
+
+    var syncContext = new DispatcherQueueSyncronizationContext();
+    SynchronizationContext.SetSynchronizationContext(syncContext);
+
+    this.ProcessEvents(); // Message pump
+});
+
+thread.SetApartmentState(ApartmentState.STA);
+thread.Start(); 
+```
+
+> Spec note: The DispatcherQueueSyncronizationContext is a new API 
+[described here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/winui3/DispatcherQueueUpdates.md)
+
+### Remarks
+
+The Window class can be used both in a UWP app or a Desktop app.
+
+In a Desktop app you can create (`new`) a Window, even if the current thread already has a Window on it.
+In a UWP app you must create a new 
+[CoreApplicationView](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationView),
+which will automatically create a new thread and a Window on it. 
+See the 'Create new Window in UWP application' example.
```

Comment:
What if I `new` a Window while on the Dispatcher for a CoreWindow? Could that create me a new [AppWindow](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.WindowManagement.AppWindow?view=winrt-19041) instead?